vendor/engines/saas/app/controllers/saas/coupons_controller.rb

```diff
@@ -11,16 +11,15 @@ def valid
     def redeem
       query = Queries::CouchCustomer.get_cust(params[:id], couch)
       doc = QueryHandler.exec(&query)
-      return render json: {success: false, message: "Couldn't find couch record: #{plan_doc.id}"} unless doc
-      plan_doc = doc
+      return render json: {success: false, message: "Couldn't find couch record: #{doc.id}"} unless doc
```

Comment:
Using `doc` alone now is :cool: but I still think we need to use `params[:id]` for the error message.